internal/wrappers/feature-flags.go

```diff
@@ -55,26 +58,70 @@ var FeatureFlagsBaseMap = []CommandFlags{
 }
 
 var FeatureFlags = map[string]bool{}
+var FeatureFlagsSpecific = map[string]bool{}
 
 func HandleFeatureFlags(featureFlagsWrapper FeatureFlagsWrapper) error {
 	allFlags, err := featureFlagsWrapper.GetAll()
 	if err != nil {
-		loadFeatureFlagsDefaultValues()
-
+		LoadFeatureFlagsDefaultValues()
 		return nil
 	}
 
 	loadFeatureFlagsMap(*allFlags)
-
 	return nil
 }
 
+func GetSpecificFeatureFlag(featureFlagsWrapper FeatureFlagsWrapper, flagName string) (*FeatureFlagResponseModel, error) {
+	if value, exists := FeatureFlagsSpecific[flagName]; exists {
+		return &FeatureFlagResponseModel{Name: flagName, Status: value}, nil
+	}
+
+	specificFlag, err := getSpecificFlagWithRetry(featureFlagsWrapper, flagName, maxRetries)
+	if err != nil {
+		UpdateSpecificFeatureFlagMapWithDefault(flagName)
+		return &FeatureFlagResponseModel{Name: flagName, Status: FeatureFlagsSpecific[flagName]}, nil
```

Comment:
can we get here NPE if we don't have a default value?

Reply:
returns false
Like before my change